alembic/templates/async/alembic.ini.mako

```diff
@@ -72,6 +72,12 @@ sqlalchemy.url = driver://user:pass@localhost/dbname
 # black.entrypoint = black
 # black.options = -l 79 REVISION_SCRIPT_FILENAME
 
+# format using "ruff" - use the exec runner, execute a binary
+# hooks = ruff
+# ruff.type = exec
+# ruff.executable = .venv/bin/ruff
```

Comment:
@zzzeek can this use %(here)s to expand the current absolute path?

Reply:
sure, it should, that woudl be nice